identity-core/src/crypto/key/key.rs

```diff
@@ -22,11 +22,11 @@ mod private {
 }
 
 // A marker type for the `Public` components of an asymmetric cryptographic key.
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Deserialize)]
 pub enum Public {}
 
 // A marker type for the `Private` components of an asymmetric cryptographic key.
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Deserialize)]
 pub enum Private {}
```

Comment:
Would an alternative to deriving `Deserialize` be to "serialize" manually to and from bytes with `Key::as_ref` and `Key::from`, since keys are just wrappers around bytes? This is not a request to change anything, just a question whether `Deserialize` has inherent benefits?

Reply:
If I understood correctly, `js_value.into_serde()` wouldn't work

Reply:
We might not need this afterall